src/code/views/node-view.tsx

```diff
@@ -265,6 +266,7 @@ export class NodeView extends React.Component<NodeViewProps, NodeViewState> {
   private lastClickLinkTime;
   private initialTitle: string;
   private node: HTMLDivElement | null;
+  private logValueLogTimeout;
```

Comment:
I don't see this value being used anywhere …

Reply:
Good catch.  I moved the code that used that into the graph view.  I've added a commit to remove the line.